### ISSUE TITLE
Tooltip repetition in column headers

### ISSUE THREAD

**Issue by marthakeezy** [msupply-foundation/open-msupply#5235]
## What went wrong? 😲

When hovering over SOH, AMC and MOS, I find the tooltip repeats the column title which is already clear enough. It should elaborate on what the acronym stands for, without repeating what is already on the screen. It looks even worse when translating those strings, where you can't turn everything into an acronym so a lot of information ends up being repeated twice.

https://github.com/user-attachments/assets/b4a332e5-7158-4385-a510-bed160874c75

## Expected behaviour 🤔

No repetition, e.g. let's remove the column title from the tooltip

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. Go to Catalogue -> Items
2. Hover over SOH, AMC or MOS -> see repetition in the tooltip

## Your environment 🌱

- Open mSupply Version: V2.3.1-RC1
- Legacy mSupply Central Server Version: V7.18.06

- Platform: Browser (Google)

- Database type: SQLite3

**Comment by mark-prins** [comment]
We show the description (if defined) and the column label as the tooltip. 
Perhaps should just show the label, if there is a description. There's little value in having a tooltip of the label when the label is already displayed too! perhaps just remove that?

**Comment by lache-melvin** [comment]
Adding on here, my least favourite flavour is when the tooltip shows a description, and then the column name again below - unnecessary at best, but often is a bit confusing/looks like a typo:

![Screenshot 2024-11-07 at 11 04 04 AM](https://github.com/user-attachments/assets/5b6bc097-12fc-4714-9c83-8643c77bbc6f)
![Screenshot 2024-11-07 at 11 06 29 AM](https://github.com/user-attachments/assets/e0760fb9-6df8-4b77-9457-5acc4a9ad342)

Occurs on non-sortable columns.

**Comment by andreievg** [comment]
Triage: Sounds like a good first issue

**Comment by lache-melvin** [comment]
FYI code to adjust is [here](https://github.com/msupply-foundation/open-msupply/blob/develop/client/packages/common/src/ui/layout/tables/components/Header/Header.tsx#L66)

**Comment by marthakeezy** [comment]
Hi team, can we please standardise the column tooltips with the rest of omSupply and between themselves?

- SOH = Stock on Hand. 1. Should 'on' be 'On'? 2. Once clicked on the item, it shows as 'Available Stock' - why not keep it Stock On Hand there too?

<img width="259" alt="Image" src="https://github.com/user-attachments/assets/5a7b9422-202a-4869-a74b-fe8afa5e4ab1" />

<img width="346" alt="Image" src="https://github.com/user-attachments/assets/6bd4835e-3d46-42f4-b07c-83a0b312e9ef" />

- AMC = Average Monthly Consumption. In the column header, shows up fine, however once clicked into the item, it just shows as AMC abbreviated? Can we keep it as 'Average Monthly Consumption' to match the rest of design?

<img width="341" alt="Image" src="https://github.com/user-attachments/assets/d2a011b0-9416-4ab3-83e2-61170de10fc7" />

- MOS = Available months of stock on hand. 1. Can we shorten it to Months Of Stock? 2. If we can't, can the initial sentence be changed to 'Available Months of Stock On Hand' to match the rest of capitalisation on the same page? 3. Once clicked into the item, it just shows as Months of Stock. 

<img width="396" alt="Image" src="https://github.com/user-attachments/assets/f20ddf42-5b2a-4bb5-b8ef-aae4497a04e7" />

<img width="348" alt="Image" src="https://github.com/user-attachments/assets/936122fd-481d-448f-bc7c-cc65850e77e4" />

**Comment by zachariah-at-msupply** [comment]
Should I make a new issue  for the above ('standardising some stock terms') and get to work on it?

**Comment by mark-prins** [comment]
> SOH = Stock on Hand. 1. Should 'on' be 'On'? 

Yeah, it makes more sense to have it as `On`. personally, I prefer to use `SoH` but accept that this (a) doesn't look as good (b) is a little pedantic and (c) that mostly we use `SOH`. When it comes to capitalising I prefer the important words only to be capitalised so I'd have an `Outbound Shipment` with an `Invoice number`. again, slightly pedantic.

Short answer - I like the little `o` in `on`

> 2. Once clicked on the item, it shows as 'Available Stock' - why not keep it Stock On Hand there too?

technically what we have is correct. We show total stock when talking about 'Stock on Hand'. The value shown in the item page is 'available stock' though; labelling as 'SOH' would be incorrect. However.. I think the use of SOH here is incorrect. My understanding is that SOH is total stock, but the figure we give for SOH is available (i.e. total - anything allocated) and the MOS is also using available. My brief research shows that everyone except the Cambridge dictionary is defining SOH as total. e.g.

![Screenshot 2024-12-30 at 9 25 21 AM](https://github.com/user-attachments/assets/74d58f5e-ac8a-46b2-a4d6-47c6dce34704)

So the question becomes 'Should we show available or total' in the item pages. I think 'total' bc this is how much stock you have in the facility and you can repurpose if you have an emergency. This means updating the heading here, and fixing up the usage for 'Available Months of Stock'

> Can we keep it as 'Average Monthly Consumption' to match the rest of design?

Done 🙏 

> 1. Can we shorten it to Months Of Stock? 

as per the second point, yes, if we change the value.

> 2. If we can't, can the initial sentence be changed to 'Available Months of Stock On Hand' to match the rest of capitalisation on the same page?

I prefer 'Months of Stock on Hand'

>  3. Once clicked into the item, it just shows as Months of Stock.
and should have been 'Available Months of Stock' but with the value change is ok.

**Comment by mark-prins** [comment]
Changes in https://github.com/msupply-foundation/open-msupply/pull/5906

**Comment by marthakeezy** [comment]
Thanks Mark!

**Comment by lache-melvin** [comment]
Closing as resolved now

**Comment by marthakeezy** [comment]
passed in V2.5.0-RC2 - moving to Done ✅
